evm/db/backends/level.py

```diff
@@ -0,0 +1,51 @@
+import shutil
+from .snapshot import Snapshot
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+    db_path = None
+
+    # Creates db as a class variable to avoid level db lock error
+    def __init__(self, db_path='./evm/db/backends/db'):
+        try:
+            import leveldb
+        except ImportError:
+            raise ImportError("LevelDB requires the leveldb \
+                               library which is not available for import.")
+        if not LevelDB.db:
+            self.db_path = db_path
+            LevelDB.db = leveldb.LevelDB(db_path,
+                                         create_if_missing=True, error_if_exists=False)
+
+    def get(self, key):
+        return bytes(self.db.Get(key))
+
+    def set(self, key, value):
+        self.db.Put(key, value)
+
+    # Returns False instead of KeyError if key doesn't exist
+    def exists(self, key):
+        return bool(self.db.Get(key, False, True, False))
```

Comment:
Can you change this function signature to keyword arguments for whatever `False, True, False` are so that those reading this code can get an idea of what those arguments mean?